charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -170,6 +175,29 @@ receivers:
     collection_interval: 20s
     auth_type: "serviceAccount"
     endpoint: "${K8S_NODE_NAME}:10250"
+  prometheus/k8s_node_cadvisor:
+    config:
+      scrape_configs:  
+      - job_name: kubernetes-cadvisor
```

Comment:
```suggestion
      - job_name: cadvisor
```

Reply:
Would you prefer to drop the `kubernetes-` prefix? I think it can be helpful to differentiate easily from perhaps similar jobs, since the user could potentially have other jobs with similar names (e.g. the API server, without prefix, you might not quickly know which API server we're referring to).

Reply:
Personal POV I prefer without the prefix, but I don't have a strong opinion about that.
If we remove the prefix we can leverage the latter without future modification of existing dashboards and rules, but not required.

Reply:
Cleared F2F, this should not be required to make it work out of the box.